src/components/views/SuggestedViews/ViewCard.tsx

```diff
@@ -0,0 +1,45 @@
+import dayjs from 'dayjs';
+import { ReactEventHandler } from 'react';
+import { ZetkinView } from 'types/zetkin';
+import { Card, CardActionArea, Fade, Grid, Typography } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    action: {
+        padding: theme.spacing(2),
+    },
+    content: {
+        minHeight: 200,
+    },
+}));
+
+type ViewCardProps = {
```

Comment:
We usually use `interface` by default, I believe? Unless using `type` is necessary (like when doing the equivalent of a `typedef`).